pages/hub/apps/guides/advanced/live/prepare-for-betting.mdx

```diff
@@ -29,14 +29,56 @@ if they cannot.
 We recommend you as a frontend operator to create a convenient interface to do that in 1 click and claim
 `1000 AZUSD` to the user's wallet.
 
-### Give a spending approval
+### Relayer fee
 
 In our live betting architecture users don't make transactions, they just sign a message that contains all the bet information and place an order.
-This order then been executed by a [Relayer](/contracts/live-relayer) - special backend service for transactions broadcasting.
+This order then been executed by a [Relayer](/contracts/live-relayer) - special backend service for transactions broadcasting. 
+
+Instead of charging a fee in native tokens when a user sends a transaction, the `Relayer` uses bet tokens as a fee.
```

Comment:
...the `Relayer` uses some of the tokens from the bet amount as a fee.

Reply:
f939912dae6e66990826b1bf2cdec72d79ecc5d8